apps/video/view.html

```diff
@@ -16,9 +16,11 @@
   <body role="application" class="skin-dark layout-fullscreen-player view-activity theme-media">
     <!-- display this at startup while we create thumbnails -->
     <div id="spinner-overlay"><div id="spinner"></div></div>
+    <div id="spinner-overlay" role="progress" data-l10n-id="spinner-overlay">
+      <div id="spinner"></div></div>
     <section role="region" id="player-view">
       <div id="video-container">
-        <video id="player" preload="metadata"></video>
+        <video id="player" preload="metadata" aria-hidden="true"></video>
```

Comment:
Recognizing I know little about ARIA, what I've read says the "aria-hidden" attribute should be set to "false" or removed when the element is presented. I don't see that happen for the elements for which 'aria-hidden="true"' has been added.